apps/addons/templates/addons/impala/details.html

```diff
@@ -92,7 +92,7 @@
 
 {# All this depends on the addon or version, and nothing needs the user,
    so we can cache it all against the addon. #}
-<section class="primary">
+<section class="primary" id="addon-description-header">
```

Comment:
I like the mantra that IDs are for humans and classes for computers, so unless this is for links to things like `amo.org/details#addon-description-header`, it's probably better to use a class.